pkg/operator/controllers/workaround/ifreload.go

```diff
@@ -17,11 +17,23 @@ import (
 const kubeNamespace = "openshift-azure-ifreload"
```

Comment:
Should this go in a const.go file along with the`pkg/operator/controllers/workaround/systemreserved.go` const vars?

Reply:
done